invenio_deposit/mappings/v6/deposits/deposit-v1.0.0.json

```diff
@@ -3,15 +3,14 @@
     "deposit-v1.0.0": {
       "properties": {
         "$schema": {
-            "type": "string",
-            "index": "not_analyzed"
+            "type": "text",
+            "index": false
         },
         "_deposit": {
           "type": "object",
           "properties": {
             "id": {
-              "type": "string",
-              "index": "not_analyzed"
```

Comment:
do we want this in the index?

Reply:
Good question. Invenio-deposit had it, so I kept it. Those changes are just to make it compatible with ElasticSearch 6.